### PR TITLE
Add definitions for criteria quality type

### PR DESCRIPTION
**Pull request by UlrikeS91**
@aeidi89 and @tgbugs, could you please check if the definitions make sense? @lzehl, asked me to add them, but I'd like some expert opinion whether or not this is 100% correct ☺️

### CODE REVIEW

**Comment by lzehl** [comment]
@aeidi89 & @tgbugs what do you think?

File: instances/criteriaQualityType/asserted.jsonld [hunk 1]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/criteriaQualityType/asserted",
   "@type": "https://openminds.ebrains.eu/controlledTerms/CriteriaQualityType",
-  "definition": null,
+  "definition": "If a criteria was 'asserted' it is based on an (expert) opinion or claim.",
```
Comment (tgbugs) [inline instances/criteriaQualityType/asserted.jsonld]:
```suggestion
  "definition": "If a criteria is 'asserted' it is based on an opinion or claim, usually made by an expert.",
```

File: instances/criteriaQualityType/processive.jsonld [hunk 2]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/criteriaQualityType/processive",
   "@type": "https://openminds.ebrains.eu/controlledTerms/CriteriaQualityType",
-  "definition": null,
+  "definition": "If a criteria was 'processive' it is based on a repeatable (automated) process.",
```
Comment (tgbugs) [inline instances/criteriaQualityType/processive.jsonld]:
```suggestion
  "definition": "If a criteria is 'processive' it is based on a repeatable and often automated process.",
```

**Review by tgbugs — COMMENTED** [review]
Suggestions.

**Review by lzehl — APPROVED** [review]

**Pull request merged**